examples/bar.jl

```diff
@@ -0,0 +1,226 @@
+using PlotlyJS
+
+
+function bar1()
+    data = bar(;x=["giraffes", "orangutans", "monkeys"],
+               y = [20, 14, 23])
+    plot(data)
+end
+
+function bar2()
+    trace1 = bar(;x = ["giraffes", "orangutans", "monkeys"],
+                 y = [20, 14, 23],
+                 name = "SF Zoo")
+    trace2 = bar(;x = ["giraffes", "orangutans", "monkeys"],
+                 y = [12, 18, 29],
+                 name = "LA Zoo")
+    data = [trace1, trace2]
+    layout = Layout(;barmode = "group")
+    plot(data,layout)
+end
+
+function bar3()
+    trace1 = bar(;x = ["giraffes", "orangutans", "monkeys"],
+                 y = [20, 14, 23],
+                 name = "SF Zoo")
+    trace2 = bar(;x = ["giraffes", "orangutans", "monkeys"],
+                 y = [12, 18, 29],
+                 name = "LA Zoo")
+    data = [trace1, trace2]
+    layout = Layout(;barmode = "stack")
+    plot(data,layout)
+end
+
+function bar4()
+    data = bar(;x = ["Product A", "Product B", "Product C"],
+                y = [20, 14, 23],
+                text = ["27% market share","24% market share", "19% market share"],
+                marker_color = "rgb(158,202,225)",
+                marker_opacity = 0.6,
+                line_color = "rgb(8,48,107)",
+                line_width = 1.5)
+    layout = Layout(;title = "January 2013 Sales Report")
+    plot(data,layout)
+end
+
+function bar5()
+    xValue = ["Product A", "Product B", "Product C"]
+    yValue = [20, 14, 23]
+
+    data = bar(;x = xValue,
+                y = yValue,
+                text = ["27% market share","24% market share", "19% market share"],
+                marker_color = "rgb(158,202,225)",
```

Comment:
Lets use `marker=attr(color="rgb(158,202,225)", opacity=0.6, ...)` here to avoid writing `marker_` many times